services/Power-BI/Whiz-tool.html

```diff
@@ -0,0 +1,248 @@
+										</li>
+										<li>
+											Quick Insights – Identify possible deviations from the recommended practices related to the model and report desig
+										</li>
+										<li>
+											Spell Check – Highlight spelling issues in the text to users
+										</li>
+										<li>
+											Single Click Documentation – Document all Power BI Model and report pages using a single click
+										</li>
+									</ul>
+									<p class="text-large">We have designed a single tool to help developers and reviewers perform model and report analysis, assisting them in delivering a consistent output that follows their organization’s standards.</p>
+									<p class="text-large">Its capabilities include:</p>
+									<ul class="text-large">
+										<li>
+											Model Analysis
+											<ul>
+												<li>Summary – Analyze the model against some of best practices and present the results in a single view</li>
+												<li>Spell Check – Identify possible spelling mistakes in table, column, and measure names</li>
+												<li>Measure Formatting – Identify the different types of measure formatting applied to the measures</li>
+												<li>DAX Formatting – Format all the DAX measures using a single click (uses <a href="https://www.nuget.org/packages/Dax.Formatter/" target="_blank">DAX Formatter nuget package</a> developed by SQLBI</li>
+												<li>Documentation – Complete documentation of the Power BI Model including all the tables, columns, measures, and relationships</li>
+												<li>Export – Export actual data from the tables present in Power BI Model for offline analysis</li>
+											  </ul>
+										</li>
+										<li>
+											Report UI Analysis
+											<ul>
+												<li>Summary – Analyze all the pages of the report; display report and page level insights to help users easily understand the different types of visuals on each report pag</li>
+												<li>Spell Check – Identify possible spelling mistakes in common components directly visible to end users (Visual Title, Axis Title, Button Text, and Textbox content)</li>
+												<li>Format Properties Summary – Compare basic format properties for all the visuals of the report in a single view (Title font properties, Axis font properties, etc.)</li>
+												<li>Visual Summary – Document columns and measures used for each visual of the report</li>
+											  </ul>
+										</li>
+									</ul>
+									<h2>Installation Steps</h2>
+									<ol class="text-large">
+										<li>
+											Download </b>Whiz Setup 2.0.0.exe</b> from the shared location (location to be confirmed).<br>	
+										</li>
+										<li>
+											Run <b>Whiz Setup 2.0.0.exe</b>on your local machine.<br>
+										</li>
+										<li>
+											If prompted about the Windows protection, click on <b>More info</b>, then click on <b>Run Anyway</b>.
+											<img src="/images/assets/Whiz-Installation.png" alt="">
```

Comment:
Add <BR> before every image. Currently the image is just below the text
Check across

Reply:
Resolved

Reply:
Resolved